src/Components/FeedbackIcon/feedbackModal.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import './feedbackIcon.css';
+import { FormattedMessage } from 'react-intl';
+import '../../Components/Results/211Button/211Button.css';
+import { useConfig } from '../Config/configHook';
+
+interface FeedbackModalProps {
+  handleCloseModal: () => void;
+}
+
+const FeedbackModal: React.FC<FeedbackModalProps> = ({ handleCloseModal }) => {
```

Comment:
To be consistent with the rest of the app.
```suggestion
const FeedbackModal = ({ handleCloseModal }: FeedbackModalProps) => {
```